.github/workflows/main.yml

```diff
@@ -7,7 +7,75 @@ on:
     - cron:  '0 22 * * *'
 
 jobs:
-  Linux-build-and-test:
+  Lint:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v2
+        with:
+          lfs: false
+      - name: Build and lint on Archlinux
+        run: | 
+          docker run \
+              --volume $PWD:/host \
+              --workdir /host/continuous-integration/linux \
+              --env "PYTHONDONTWRITEBYTECODE=1" \
+              archlinux/base \
+              bash -c "./setup.sh && ./build.sh && ./lint.sh"
+
+  Create-source-dist:
```

Comment:
How about `create-source-distribution`?

Reply:
Sure. I'll change all the other "dist" to "distribution" too then.